android/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java

```diff
@@ -315,8 +315,8 @@ private String readAssetStream(InputStream stream) {
    * @return
    */
   private WebResourceResponse handleProxyRequest(WebResourceRequest request, PathHandler handler) {
-    try {
-      final String method = request.getMethod();
+    final String method = request.getMethod();
+    if (method.equals("GET")) try {
```

Comment:
Looks good, but add curly braces to the if and put the try in next line

Something like 
```
if (method.equals("GET")) { 
  try { 
    //code
  } catch () {
    //code
  }
}
```

Reply:
@jcesarmobile oh sure, now fixed.